actions.go

```diff
@@ -53,22 +53,27 @@ const (
 
 // New creates a new wrapper with helpers for outputting information in GitHub
 // actions format.
-func New() *Action {
-	return &Action{w: os.Stdout}
+func New(opts ...Option) *Action {
+	a := &Action{w: os.Stdout, getenv: os.Getenv}
+	for _, opt := range opts {
+		opt(a)
```

Comment:
Can we add a guard that `opt != nil` before calling? Sometimes developers do weird things...

Reply:
Sure thing. Done.